templates/platform_landing_zone/examples/full-multi-region/hub-and-spoke-vnet.tfvars

```diff
@@ -246,282 +246,245 @@ management_group_settings = {
         }
       }
     }
-    /*
-    # Example of how to update a policy assignment enforcement mode for DDOS Protection Plan
-    connectivity = {
-      policy_assignments = {
-        Enable-DDoS-VNET = {
-          enforcement_mode = "DoNotEnforce"
-        }
-      }
-    }
-    landingzones = {
-      policy_assignments = {
-        Enable-DDoS-VNET = {
-          enforcement_mode = "DoNotEnforce"
-        }
-      }
-    }
-    */
-    /*
-    # Example of how to update a policy assignment enforcement mode for Private Link DNS Zones
-    corp = {
-      policy_assignments = {
-        Deploy-Private-DNS-Zones = {
-          enforcement_mode = "DoNotEnforce"
-        }
-      }
-    }
-    */
-  }
-  /*
```

Comment:
We can retain the role assignment examples.

Reply:
sorry - done